src/Core_Fix_ResourceUnloadOptimizations/Core.ResourceUnloadOptimizations.cs

```diff
@@ -16,19 +18,38 @@ public partial class ResourceUnloadOptimizations
         private static Func<AsyncOperation> _originalUnload;
 
         private static int _garbageCollect;
+        private static readonly List<Func<bool>> IsLoadingChecks = new List<Func<bool>>
+        {
+            GetIsNowLoadingFade
```

Comment:
Instead of a list it would be faster and cleaner to just write it out in code since there's only 2 items at most.

Reply:
I thought so too, but I was using the profiler to verify and didn't seem to be the case (but I see the profiler had a bug totaling things, so maybe that's why).  I'll simplify it back down.